.github/.jira_sync_config.yaml

```diff
@@ -16,7 +16,9 @@ settings:
     opened: Triaged
     closed: done
   label_mapping:
-    enhancement: Story
-    bug: Bug
-    spike: Spike
-    epic: Epic
+    "Type: Enhancement": Story
+    "Type: Bug": Bug
+    "Type: Research": Spike
+    "Type: Documentation": Documentation
+    "Type: Cleanup": Task
+    "Type: Refactor": Task
```

Comment:
`Type: Question` and `Type: Invalid` labels are missing, is that on purpose? (same question for other similar PRs submitted in other projects)

Reply:
It's on purpose because those labels are for unactionable tickets, so we shouldn't need a Jira task for them.

Reply:
Oh yeah my bad -_-